stores/useWalletStore.tsx

```diff
@@ -40,13 +40,16 @@ import { mapFromEntries, mapEntries } from '../tools/core/script'
 import { GoverningTokenType } from '../models/enums'
 import { AccountInfo, MintInfo } from '@solana/spl-token'
 import tokenService from '@utils/services/token'
+import { EndpointTypes } from '@models/types'
+
+export interface ConnectionContext {
+  cluster: EndpointTypes
+  current: Connection
+  endpoint: string
+}
 interface WalletStore extends State {
   connected: boolean
-  connection: {
-    cluster: string
-    current: Connection
-    endpoint: string
-  }
```

Comment:
making this object its own interface to be used elsewhere

Reply:
ultimately i plan to make a bigger refactor to potentially unify connectionContext and rpcContext but pull conectionContext out of WalletStore into its own state so it can be used separately since not all cases require a wallet

Reply:
yeah, that's the right direction